README.md

```diff
@@ -176,8 +181,14 @@ Options:
   -t, --tokens <TOKENS>                  [env: TOKENS=]
 ```
 
+## Metrics
+
+| :exclamation: | To see SDK usage metrics from Edge instances, you will need Unleash v4.22 or newer. |
+| ------------- | ----------------------------------------------------------------------------------- |
```

Comment:
I take it this is supposed to be an admonition, right? I would do something like this if you want to use it call it out. Using a table seems semantically very dubious. But also where do you "see metrics"? Is that in the network view? Or doesn't Unleash process it at all before 4.22? (Also, not everyone can see 💁🏼)

But also: is there nothing more to say about metrics? It looks like it's missing a description? Does edge report metrics? How does it work? I suggest putting a paragraph or two after the admonition to explain how edge works with metrics.

```suggestion
**❗  Note:**  For Unleash to correctly register SDK usage metrics sent from Edge instances, your Unleash instance must be v4.22 or newer.
```

Reply:
Agreed, will update. Thanks for the feedback. :)